assets/js/vue/pages/PersonProfile.vue

```diff
@@ -0,0 +1,111 @@
+<template>
+    <div v-show="showProfile" class="container">
+        <div class="main-body">
+            <div class="row gutters-sm">
+                <div class="col-md-4 mb-3">
+                    <AvatarCard :person-details="personProfileData"/>
+                </div>
+                <div class="col-md-8">
+                    <UserDetailsCard :person-details="personProfileData"/>
+                </div>
+            </div>
+        </div>
+    </div>
+</template>
+
+<script>
+import AvatarCard from "@/vue/components/person-profile/AvatarCard";
+import UserDetailsCard from "@/vue/components/person-profile/UserDetailsCard";
+const axios = require('axios');
+
+export default {
+    name: "PersonProfile",
+    components: { AvatarCard, UserDetailsCard },
+    props: {
+        personId: {
+            type: Number
+        }
+    },
+    data() {
+        return {
+            showProfile: false,
+            personProfileData: {},
+            avatarInfo: {}
+        }
+    },
+    mounted() {
+        let loader = null;
+        let thisComponent = this;
+        const axiosInstance = axios.create({});
+        axiosInstance.interceptors.request.use(function (config) {
+            loader = thisComponent.$loading.show({
+                container: thisComponent.$refs.formContainer,
+                loader: 'bars',
+                color: '#007bff',
+            });
+            return config;
+        }, function (error) {
+            return Promise.reject(error);
+        });
+
+        function hideLoader(){
+            loader && loader.hide();
+            loader = null;
+        }
+
+        axiosInstance.interceptors.response.use(function (response) {
+            hideLoader();
+            return response;
+        }, function (error) {
+            hideLoader();
+            return Promise.reject(error);
+        });
+
+        axiosInstance
+            .get(Routing.generate('pelagos_api_get_person') + "/" + this.personId)
+            .then(response => {
+                this.personProfileData = response.data;
+                this.showProfile = true;})
+            .catch(error => {
+                console.log(error);
+                this.showProfile = false;
+            });
```

Comment:
I don't know if this is a style choice, but we should pick either one tab over like this, or what we have done before, it have this one tab back.

Reply:
I looked up the Eslint rules for promise chaining, they say that new line for each chained call